core/dungeon-masters-guide/items/items-wondrous.xml

```diff
@@ -4085,4 +4085,320 @@
+				<tr><td>71-80</td><td>As 61-70 above, except the spell is 2nd level.</td></tr>
+				<tr><td>81-90</td><td>As 61-70 above, except the spell is 3rd level.</td></tr>
+				<tr><td>91-00</td><td>While attuned to the artifact, you gain a +1 bonus to Armor Class.</td></tr>
+			</table>
+
+			<h4>Minor Detrimental Properties</h4>
+			<table>
+				<thead>
+					<tr><td>d100</td><td>Property</td></tr>
+				</thead>
+				<tr><td>01-05</td><td>While attuned to the artifact, you have disadvantage on saving throws against spells.</td></tr>
+				<tr><td>06-10</td><td>The first time you touch a gem or piece of jewelry while attuned to this artifact, the value of the gem or jewelry is reduced by half.</td></tr>
+				<tr><td>11-15</td><td>While attuned to the artifact, you are blinded when you are more than 10 feet away from it.</td></tr>
+				<tr><td>16-20</td><td>While attuned to the artifact, you have disadvantage on saving throws against poison.</td></tr>
+				<tr><td>21-30</td><td>While attuned to the artifact, you emit a sour stench noticeable from up to 10 feet away.</td></tr>
+				<tr><td>31-35</td><td>While attuned to the artifact, all holy water within 10 feet of you is destroyed.</td></tr>
+				<tr><td>36-40</td><td>While attuned to the artifact, you are physically ill and have disadvantage on any ability check or saving throw that uses Strength or Constitution.</td></tr>
+				<tr><td>41-45</td><td>While attuned to the artifact, your weight increases by 1d4 × 10 pounds.</td></tr>
+				<tr><td>46-50</td><td>While attuned to the artifact, your appearance changes as the DM decides.</td></tr>
+				<tr><td>51-55</td><td>While attuned to the artifact, you are deafened when you are more than 10 feet away from it.</td></tr>
+				<tr><td>56-60</td><td>While attuned to the artifact, your weight drops by 1d4 × 5 pounds.</td></tr>
+				<tr><td>61-65</td><td>While attuned to the artifact, you can't smell.</td></tr>
+				<tr><td>66-70</td><td>While attuned to the artifact, nonmagical flames are extinguished within 30 feet of you.</td></tr>
+				<tr><td>71-80</td><td>While you are attuned to the artifact, other creatures can't take short or long rests while within 300 feet of you.</td></tr>
+				<tr><td>81-85</td><td>While attuned to the artifact, you deal 1d6 necrotic damage to any plant you touch that isn't a creature.</td></tr>
+				<tr><td>86-90</td><td>While you are attuned to the artifact, animals within 30 feet of you are hostile toward you.</td></tr>
+				<tr><td>91-95</td><td>While attuned to the artifact, you must eat and drink six times the normal amount each day.</td></tr>
+				<tr><td>96-00</td><td>While you are attuned to the artifact, your flaw is amplified in a way determined by the DM.</td></tr>
+			</table>
+
```

Comment:
separately, i want to mention that tables are in the wrong order (which makes it annoying to check, but also feels weird that Minor effects go after Major ones), if we follow order in DMG it goes like this:
• Minor Beneficial
• Major Beneficial
• Minor Detrimental
• Major Detrimental